draft-ietf-httpbis-messaging-latest.xml

```diff
@@ -1132,7 +1132,8 @@ https://www.example.org
    framing is invalid and the recipient &MUST; treat it as an unrecoverable
    error, unless the field value can be successfully parsed as a
    comma-separated list (<xref target="abnf.extension"/>), all values in the
-   list are valid, and all values in the list are the same.
+   list are valid, and all values in the list are the same when compared
+   numerically.
```

Comment:
Hm, that's a change from RFC 7230. Do we really need to do that? If so, we need to mention it in the Changes section.

Reply:
I think we do; will do.

Reply:
I reverted that, since it's not clear what interoperability really is here.